consenlabs/tokenlon-v5-subgraph
# [WIP] Migrate uniswap price

Migrate Uniswap token price in to Tokenlon

URI: https://thegraph.com/explorer/subgraph/sc0vu/tokenlonv5?version=pending

How to test:
```
$ SUBGRAPH_REPO=yourepo npm run deploy-staging
```

Before deploy the test, make sure the `generated` was updated
```
$ npm run codegen
```

The syncing time on subgraph is quite slow (10.3m blocks / 10hours)
